utility/util.js

```diff
@@ -0,0 +1,34 @@
+function findOverlap(country1, country2){
+    var countryOne = country1['noVisa']
+    var countryTwo = country2['noVisa']
+    var overlap = []
+    for (i=0; i<countryOne.length; i++){
+        countryTwo.includes(countryOne[i]) ? overlap.push(countryOne[i]) : null
```

Comment:
not sure why you're using a turnery here...

Reply:
How else did you want me to check for the overlap?

Reply:
```
if(countryTwo.includes(countryOne[i])){
  overlap.push(countryOne[i])
}